helm-charts/falcon-sensor/templates/container_secret.yaml

```diff
@@ -12,7 +12,7 @@ data:
 type: kubernetes.io/dockerconfigjson
 {{- if or .Values.container.image.pullSecrets.namespaces .Values.container.image.pullSecrets.allNamespaces }}
 {{- $name := ( .Values.container.image.pullSecrets.name | default (printf "%s-pull-secret" (include "falcon-sensor.fullname" .))) }}
-{{- $myns := split "," .Values.container.image.pullSecrets.namespaces -}}
+{{- $myns := split "," .Values.container.image.pullSecrets.namespaces | default "" -}}
```

Comment:
If _namespace_ is not provided and creating this resource in _default_ namespace, would it be in sync with other resources created by this helm chart instantiation?
(i,e) would other resources also be created in _default_ namespace?

Reply:
Not sure what you are asking here, these settings are for deploying the secret to the non-deployment namespaces when the injector is used.

Reply:
ACK. 
That clarified my query. I thought that this change was for the `injector` itself.